x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/license_callout/constants.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const LICENSE_CALLOUT_HEADING = i18n.translate(
+  'xpack.enterpriseSearch.licenseCalloutHeading',
+  {
+    defaultMessage: 'Enterprise-grade functionality for teams big and small',
```

Comment:
[Not blocking] This is the same copy from the Setup Guide CTA:

https://github.com/elastic/kibana/blob/f3399620cf558efe1e3a862e32289839f88ed815/x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/setup_guide/setup_guide_cta.tsx#L24-L26

Should we consider starting a top level `enterprise_search/constants.ts` file and DRY out a single string for re-use? 🤷 Not a change request, just curious about it

Reply:
Yeah, I totally thought about that but forgot to actually implement it. Will throw up a commit. Nice catch!

Reply:
https://github.com/elastic/kibana/pull/99122/commits/cff2a30810f2b70728b68e529881c7591199ee0b